Index: init.h
===================================================================
--- init.h.orig	2007-03-21 21:33:20.000000000 +0100
+++ init.h	2007-03-21 21:35:32.000000000 +0100
@@ -19,7 +19,7 @@
 
 #ifdef _MAKEDOC
 # include "config.h"
-# include "makedoc-defs.h"
+/* # include "makedoc-defs.h" include only documented options for Debian */
 #else
 # include "sort.h"
 #endif
Index: functions.h
===================================================================
--- functions.h.orig	2007-03-21 21:35:37.000000000 +0100
+++ functions.h	2007-03-21 21:35:51.000000000 +0100
@@ -35,7 +35,7 @@
 
 #ifdef _MAKEDOC
 # include "config.h"
-# include "makedoc-defs.h"
+/* # include "makedoc-defs.h" include only documented options for Debian */
 #endif
 
 struct binding_t OpGeneric[] = { /* map: generic */
