--- po/Makefile.in.in.orig
+++ po/Makefile.in.in
@@ -85,7 +85,7 @@
 # otherwise packages like GCC can not be built if only parts of the source
 # have been downloaded.
 
-$(srcdir)/$(PACKAGE).pot: $(POTFILES) $(srcdir)/POTFILES.in
+$(srcdir)/$(PACKAGE).pot: #$(POTFILES) $(srcdir)/POTFILES.in
 	rm -f $@
 	$(XGETTEXT) --default-domain=$(PACKAGE) --directory=$(top_srcdir) \
 	  --add-comments --keyword=_ --keyword=N_ \
