Index: Muttrc.head.in
===================================================================
--- Muttrc.head.in.orig	2007-03-01 23:38:41.396146688 +0100
+++ Muttrc.head.in	2007-03-02 00:19:31.107733920 +0100
@@ -6,6 +6,33 @@
 #
 ignore "from " received content- mime-version status x-status message-id
 ignore sender references return-path lines
+ignore date delivered-to precedence errors-to in-reply-to user-agent
+ignore x-loop x-sender x-mailer x-msmail-priority x-mimeole x-ms- x-priority
+ignore x-accept-language x-authentication-warning thread- priority importance
+ignore x-original-to domainkey-signature dkim-signature
+
+# emacs-like bindings
+bind editor    "\e<delete>"    kill-word
+bind editor    "\e<backspace>" kill-word
+
+# map delete-char to a sane value
+bind editor     <delete>  delete-char
+
+# some people actually like these settings
+#set pager_stop
+#bind pager <up> previous-line
+#bind pager <down> next-line
+
+# Specifies how to sort messages in the index menu.
+set sort=threads
+
+# Uncomment if your MTA does not strip Bcc: headers.
+# (exim4 and postfix strip them, exim(3) does not.)
+#unset write_bcc
+# Postfix and qmail use Delivered-To for detecting loops
+unset bounce_delivered
+
+set mixmaster="mixmaster-filter"
 
 # imitate the old search-body function
 macro index \eb "<search>~b " "search in message bodies"
@@ -15,14 +42,16 @@ macro index,pager \cb "<pipe-message> ur
 macro attach,compose \cb "<pipe-entry> urlview<Enter>" "call urlview to extract URLs out of a message"
 
 # Show documentation when pressing F1
-macro generic,pager <F1> "<shell-escape> less @docdir@/manual.txt<Enter>" "show Mutt documentation"
+macro generic <f1> "!zcat @docdir@/mutt/manual.txt.gz | sensible-pager\n" "Show Mutt documentation"
+macro index   <f1> "!zcat @docdir@/mutt/manual.txt.gz | sensible-pager\n" "Show Mutt documentation"
+macro pager   <f1> "!zcat @docdir@/mutt/manual.txt.gz | sensible-pager\n" "Show Mutt documentation"
 
 # show the incoming mailboxes list (just like "mutt -y") and back when pressing "y"
 macro index,pager y "<change-folder>?<toggle-mailboxes>" "show incoming mailboxes list"
 bind browser y exit
 
 # If Mutt is unable to determine your site's domain name correctly, you can
-# set the default here.
+# set the default here. (better: fix /etc/mailname)
 #
 # set hostname=cs.hmc.edu
 
@@ -84,7 +113,6 @@ attachments   +I text/plain
 attachments   -A message/external-body
 attachments   -I message/external-body
 
-##
-## More settings
-##
+# enable mime lookup by extension
+mime_lookup application/octet-stream
 
