--- init.c.orig
+++ init.c
@@ -2929,7 +2929,7 @@
   {
     p = getenv ("EDITOR");
     if (!p)
-      p = "vi";
+      p = "/usr/bin/editor";
   }
   Editor = safe_strdup (p);
   Visual = safe_strdup (p);
--- init.h.orig
+++ init.h
@@ -551,7 +551,7 @@
   ** .pp
   ** This variable specifies which editor is used by mutt.
   ** It defaults to the value of the VISUAL, or EDITOR, environment
-  ** variable, or to the string "vi" if neither of those are set.
+  ** variable, or to the string "/usr/bin/editor" if neither of those are set.
   */
   { "encode_from",	DT_BOOL, R_NONE, OPTENCODEFROM, 0 },
   /*
