Index: debian-mutt/Makefile.in
===================================================================
--- debian-mutt.orig/Makefile.in	2007-06-14 10:47:56.000000000 +0200
+++ debian-mutt/Makefile.in	2007-06-14 10:51:17.000000000 +0200
@@ -73,10 +73,10 @@ am__objects_1 = patchlist.$(OBJEXT)
 am_mutt_OBJECTS = $(am__objects_1) addrbook.$(OBJEXT) alias.$(OBJEXT) \
 	attach.$(OBJEXT) base64.$(OBJEXT) browser.$(OBJEXT) \
 	buffy.$(OBJEXT) color.$(OBJEXT) crypt.$(OBJEXT) \
-	cryptglue.$(OBJEXT) commands.$(OBJEXT) complete.$(OBJEXT) \
-	compose.$(OBJEXT) compress.$(OBJEXT) copy.$(OBJEXT) curs_lib.$(OBJEXT) \
-	curs_main.$(OBJEXT) date.$(OBJEXT) edit.$(OBJEXT) \
-	enter.$(OBJEXT) flags.$(OBJEXT) init.$(OBJEXT) \
+	cryptglue.$(OBJEXT) compress.$(OBJEXT) commands.$(OBJEXT) \
+	complete.$(OBJEXT) compose.$(OBJEXT) copy.$(OBJEXT) \
+	curs_lib.$(OBJEXT) curs_main.$(OBJEXT) date.$(OBJEXT) \
+	edit.$(OBJEXT) enter.$(OBJEXT) flags.$(OBJEXT) init.$(OBJEXT) \
 	filter.$(OBJEXT) from.$(OBJEXT) getdomain.$(OBJEXT) \
 	group.$(OBJEXT) handler.$(OBJEXT) hash.$(OBJEXT) \
 	hdrline.$(OBJEXT) headers.$(OBJEXT) help.$(OBJEXT) \
Index: debian-mutt/config.h.in
===================================================================
--- debian-mutt.orig/config.h.in	2007-06-14 10:47:56.000000000 +0200
+++ debian-mutt/config.h.in	2007-06-14 10:51:11.000000000 +0200
@@ -513,7 +513,7 @@
 /* Define to enable Sun mailtool attachments support. */
 #undef SUN_ATTACHMENT
 
-/* Define to enable compressed mailboxes support */
+/* Define to support compressed folders. */
 #undef USE_COMPRESSED
 
 /* Define to use dotlocking for mailboxes. */
Index: debian-mutt/configure
===================================================================
--- debian-mutt.orig/configure	2007-06-14 10:47:56.000000000 +0200
+++ debian-mutt/configure	2007-06-14 10:51:11.000000000 +0200
@@ -1367,11 +1367,11 @@ Optional Features:
   --enable-nfs-fix        Work around an NFS with broken attributes caching
   --enable-mailtool       Enable Sun mailtool attachments support
   --enable-locales-fix    The result of isprint() is unreliable
+  --enable-compressed     Enable compressed folders support
   --enable-exact-address  Enable regeneration of email addresses
   --enable-hcache         Enable header caching
   --disable-iconv         Disable iconv support
   --disable-nls           Do not use Native Language Support
-  --enable-compressed     Enable compressed folders support
 
 Optional Packages:
   --with-PACKAGE[=ARG]    use PACKAGE [ARG=yes]
@@ -15106,9 +15106,9 @@ _ACEOF
 fi
 
 
-# Check whether --enable-compressed or --disable-compressed was given.
+# Check whether --enable-compressed was given.
 if test "${enable_compressed+set}" = set; then
-  enableval="$enable_compressed"; if test x$enableval = xyes; then
+  enableval=$enable_compressed; if test x$enableval = xyes; then
 
 cat >>confdefs.h <<\_ACEOF
 #define USE_COMPRESSED 1
@@ -15118,6 +15118,7 @@ _ACEOF
 fi
 
 
+
 # Check whether --with-exec-shell was given.
 if test "${with_exec_shell+set}" = set; then
   withval=$with_exec_shell; if test $withval != yes; then
