--- doc/Makefile.in.orig
+++ doc/Makefile.in
@@ -182,9 +182,9 @@
 	$(EDIT) $(srcdir)/mutt.man > $@
 
 stamp-doc-xml: $(top_srcdir)/makedoc.c $(top_srcdir)/init.h manual.xml.head manual.xml.tail \
-               $(top_srcdir)/VERSION $(top_srcdir)/reldate.h
+               $(top_srcdir)/VERSION $(top_builddir)/reldate.h
 	$(MAKE) ../makedoc$(EXEEXT)
-	( sed -e "s/@VERSION\@/`cat $(top_srcdir)/VERSION` (`cut -d\\" -f2 $(top_srcdir)/reldate.h`)/" $(srcdir)/manual.xml.head ;\
+	( sed -e "s/@VERSION\@/`cat $(top_srcdir)/VERSION` (`cut -d\\" -f2 $(top_builddir)/reldate.h`)/" $(srcdir)/manual.xml.head ;\
 	  $(MAKEDOC_CPP) $(top_srcdir)/init.h | ../makedoc -s ) |      \
 		cat - $(srcdir)/manual.xml.tail > manual.xml
 	touch stamp-doc-xml
